static/app/utils/profiling/platforms.tsx

```diff
@@ -85,7 +85,7 @@ export function getDocsPlatformSDKForPlatform(
     return 'javascript-solid';
   }
 
-  if (platform === 'dart-flutter') {
+  if (platform === 'dart-flutter' || platform === 'dart') {
```

Comment:
By creating a project with "dart," we get a "dart" platform instead of "dart-flutter." We need to investigate this further, but this code resolves the issue of the doc not being visible to users.

Reply:
@JonasBa can you please confirm this?

Reply:
will move this code to a separate PR